Testshell/test_script_command.cpp

```diff
@@ -1,9 +1,21 @@
 #include "command.h"
 #include "test_shell.h"
 
+bool checkPartialWriteSuccess(TestShell *shell, int lba,
+                                    unsigned long value) {
+  shell->getSSD()->read(lba);
+
+  string valueStr = convertHexToString(value);
+  if (shell->getSSD()->getResult() == "ERROR" ||
+      shell->getSSD()->getResult() != valueStr) {
+    return false;
+  }
+  return true;
+}
+
 void TestScript1::execute(TestShell* shell, const Command& command) {
-  string valueStr = command.args[0];
-  unsigned long value = std::stoul(valueStr, nullptr, shell->getHexBase());
+  unsigned long value = 0xAAAABBBB;
+  string valueStr = convertHexToString(value);
 
```

Comment:
command 의 args 가 없는 지 확인해주는 guard checker 하나 추가하면 좋을 거 같습니다.

```suggestion
  if(args.size() != 0){
    std::cout << "INVALID_COMMAND\n";
    return;
  }
  unsigned long value = 0xAAAABBBB;
  string valueStr = convertHexToString(value);

```

Reply:
좋은 의견 감사합니다 수정하겠습니다